### ISSUE TITLE
Multi-Datacenter Replication connection failure

### ISSUE THREAD

**Issue by pma** [basho/riak_kv#1780]
Hi,

I tried setting up replication between two clusters using the guide at https://docs.riak.com/riak/kv/latest/configuring/v3-multi-datacenter/quick-start/index.html.

Cluster1 is running 2.9.8 and is the source. Cluster2 is running 3.0.1 (newly formed with no data).

After setting the clustername in both clusters I try connection from source to sink and get the confusing message below:

```
riak_core_cluster_mgr:is_ok_to_connect:481 ClusterManager: dropping connection {cluster_by_name,"Cluster2"} to identically named cluster: "Cluster1"
```

In nodes of both source and sink clusters, TCP port 9080 is accessible from all nodes of the other cluster.

Any ideas on what could be the problem would be really appreciated.

**Comment by martinsumner** [comment]
I don't know what is happening, but to clear up the confusion in the log message - my understanding what it means to say is:

"My name is Cluster1, I have a remote connection setup to Cluster2, but I have now received an update from Cluster2 to say its name is now Cluster1 - but that is my name" 

The tutorial looks fine to me.  Unless you've done something like join the two clusters together?  Perhaps check by looking at the members in `riak admin status` that they are still two separate clusters.

**Comment by pma** [comment]
@martinsumner Thanks for taking the time to reply. Merry Christmas!

After reading the riak-repl settings again (https://docs.riak.com/riak/kv/2.2.3/configuring/v3-multi-datacenter.1.html#riak-repl-settings), I noticed I probably had to change the default configuration for the cluster manager IP:PORT, which is 0.0.0.0:9080.

In /etc/riak/riak.conf I changed the mdc.cluster_manager property from 0.0.0.0:9080 to the actual IP of the node.
Did the same for the `cluster_mgr` property in /etc/riak/advanced.config.

After restarting all nodes in both clusters I was able to establish the connection and start a fullsync.

I assumed incorrectly that the "0.0.0.0:9080" was simply specifying the bind interface and listen port for the cluster manager, as is the case with other listen ports of riak. But apparently there is more to it, as the IP seems to be advertised to other nodes, as part of the repl protocol, so they can't all be 0.0.0.0.

Am I wrong to assume that the default 0.0.0.0 won't work in any situation? Maybe it makes more sense to force this to be changed, if the multi datacenter replication is enabled.
